### PR TITLE
Feature/confirmation page

### LINKED ISSUE #24: Reviews on Brewery Details disappear when refreshed
Issue might be that needing to put in componentDidUpdate

### PR DESCRIPTION
**Pull request by aominhlong**
# Description
Fixed the bug where the brewery details were not rendering properly on refresh. Refactor files to remove unused code.  

Were there any issues that arose? 
N/A

Is there anything you need from your teammates? 
N/A

Any other comments, questions, or concerns?
N/A

## Issues
Closes #24 

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactor feature (non-breaking change that modifies existing work)
- [ ] Cosmetic changes (changes to layout or visuals in CSS file)

# How Has This Been Tested?
- [x] React Dev Tools
- [x] Functionality check through web browser
- [ ] Ran Cypress test suite

# Checklist:
- [x] My code follows the style guidelines of this project (semicolons, indentation, naming conventions, etc..).
- [x] I have performed a self-review of my code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have deleted all comments after functionality and understanding is final.
- [x] I have added any issues closed by this PR in the Issues section.
- [x] I have added others as reviewers.

### CODE REVIEW

File: src/Components/App.tsx [hunk 1]
```diff
@@ -88,22 +89,35 @@ class App extends React.Component<{}, State> {
...
+    return fetch('http://localhost:3001/api/v1/reviews')
       .then(res => res.json())
       .then(data => {
-        console.log('data from fetch', data)
-        this.setState({reviews: data})}
+        console.log('data: ', data)
+        return data}
         )
-    const filteredData = this.state.reviews.filter(review => {
+  }
```
Comment (rachellhill) [inline src/Components/App.tsx:104]: This looks great!

**Review by rachellhill — APPROVED** [review]
Great work fixing this bug!

File: src/Components/BreweryDetails.tsx [hunk 2]
```diff
@@ -59,7 +58,7 @@ class BreweryDetails extends React.Component<DetailsProps, DetailsState> {
                     <p className='address'>Address: {`${this.state.brewery.city}, ${this.state.brewery.state} ${this.state.brewery.postal_code}`}</p>
                   </div>
                 </div>
-                <ReviewForm refresh={this.props.refresh} filteredReviews={this.props.filteredReviews} reviews={this.props.reviews} id={this.state.brewery.id}/>
+                  <ReviewForm refresh={this.props.refresh} filteredReviews={this.props.filteredReviews} reviews={this.props.reviews} id={this.props.id}/>
```
Comment (stephanie-roe) [inline src/Components/BreweryDetails.tsx:61]: BUG SQUASHED

**Review by stephanie-roe — APPROVED** [review]
Love the confirmation page and sooo happy we were able to finally squash that bug in our code.

File: src/Components/ConfirmationPage.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
```
Comment (userigorgithub) [inline src/Components/ConfirmationPage.tsx:1]: GJ implementing confirmation gif!!!

File: src/Components/BreweryDetails.tsx (hunk 2 above)
Comment (userigorgithub) [inline src/Components/BreweryDetails.tsx:61]: this.props.id was a key to fixing the bug it seems.

**Review by userigorgithub — APPROVED** [review]
Great job all. Implemented confirmation gif and fixed the bug!

**Pull request merged**